### ISSUE TITLE
URGENT

### ISSUE THREAD

**Issue by carolinehchristopher** [chalkclassroom/CoachingApp#624]
@thompchr 
I logged into chalk coaching (regular website, not 2nd environment) to prep for a coach training I’m doing this afternoon using my admin account, and I tried conducting an observation, and even though the Results snapshot is accurate, it takes me to old results when I click view results.  In the date dropdown, it doesn’t show that I took any data today.
I also tried a regular coach account and had the same issue.  
I also tried exporting, and nothing happens. 
This is an urgent issue since I have an appointment to train a new coach on the app today.  Can you help?

**Comment by carolinehchristopher** [comment]
Sure!
Caroline Christopher is inviting you to a scheduled Zoom meeting.

Topic: Caroline Christopher's Personal Meeting Room

Join Zoom Meeting
https://vanderbilt.zoom.us/j/6311722185

Meeting ID: 631 172 2185
One tap mobile
+16465588656,,6311722185# US (New York)
+13462487799,,6311722185# US (Houston)

Dial by your location
        +1 646 558 8656 US (New York)
        +1 346 248 7799 US (Houston)
Meeting ID: 631 172 2185
Find your local number: https://vanderbilt.zoom.us/u/aznZZ45IN

Join by SIP
6311722185@zoomcrc.com

Join by H.323
162.255.37.11 (US West)
162.255.36.11 (US East)
115.114.131.7 (India Mumbai)
115.114.115.7 (India Hyderabad)
213.19.144.110 (Amsterdam Netherlands)
213.244.140.110 (Germany)
103.122.166.55 (Australia Sydney)
103.122.167.55 (Australia Melbourne)
149.137.40.110 (Singapore)
64.211.144.160 (Brazil)
149.137.68.253 (Mexico)
69.174.57.160 (Canada Toronto)
65.39.152.160 (Canada Vancouver)
207.226.132.110 (Japan Tokyo)
149.137.24.110 (Japan Osaka)
Meeting ID: 631 172 2185

**Comment by areyn77463** [comment]
aaron.reynolds@maxxpotential.com

**Comment by MattHaslemTech** [comment]
matt.haslem@maxxpotential.com

**Comment by carolinehchristopher** [comment]
@areyn77463 @MattHaslemTech @thompchr any update on this?  I am holding my next coach training on Tuesday starting at 9am, so it will be really important for the website to work at that point so they can practice conducting observations and looking at the results that are generated.

**Comment by MattHaslemTech** [comment]
@carolinehchristopher I'm really sorry we weren't able to fix it sooner but we fixed it to the point where the observation data is being saved and displayed correctly. 

Unfortunately, I'm still experiencing the issue where it'll show a previous observation if you click 'show results' too fast. But the rest of the functionality should be up and running now

**Comment by carolinehchristopher** [comment]
@MattHaslemTech @areyn77463 thank you for the update!  Super glad to know that the data is being saved and displayed correctly.

**Comment by areyn77463** [comment]
@carolinehchristopher Exporting was doing nothing because there was no data to pull. If you set the start data to before the most recent observation that was saved it works as intended. Currently, the most recent observation for most types is September 8th and some of them go as far back as June or July. Now that new data  is being saved you should run into this problem less and less.

Also, did you possibly receive an email about a password reset for my email address? I tried to reset my password through firebase because I did not save it before our meeting ended. I do not need it anymore currently but it may be helpful to have for any admin problems in the future.

**Comment by carolinehchristopher** [comment]
I didn't see a password reset email, but I think Ilknur is able to generate one for you. Keep a lookout

**Comment by carolinehchristopher** [comment]
@areyn77463 just let me know if it didn't come through, and we'll figure out how to get another admin account for you.  Have a great weekend!

**Comment by areyn77463** [comment]
@carolinehchristopher - @MattHaslemTech and I had some questions about the users page. We left some comments on pages 34, 35, 38 of the mockup (https://docs.google.com/presentation/d/1sOKMX8CHZP7AQJcU-d9Wjmk5tHiuGg_V6d11n-Tas0A/edit#slide=id.gff2244bdcc_0_1136). Would you mind addressing these at your earliest convenience?

**Comment by carolinehchristopher** [comment]
@areyn77463 @MattHaslemTech I am meeting with the team at 2pm, and I'd like to go through your questions with them before I reply if that's okay.  Just want to make sure I get it right

**Comment by carolinehchristopher** [comment]
@areyn77463 @MattHaslemTech Okay, my head is going to explode, but a lot came up in the meeting.  I'm wondering if I could get a meeting with you (either or both) on Monday to discuss.  I'd ask for something sooner, but I have to travel for a conference tomorrow through Friday.  Ilknur and I are both free Monday at or after 10am.

Basically, we realized we're unsure about how the actual coach user account and the admin portal will talk to one another/share information.

We did a test, and right now, if a coach adds a teacher to their list (name and email address), it doesn't automatically generate a teacher user account for that person -- it just links a teacher to a coach.  If I then go in and create a teacher user account using that same teacher name and email address, it allows me to create an account, but it has a separate id in BigQuery from the one that was generated when a coach adds a teacher.  

So then, what are the implications of giving the admin portal the ability to add or edit user information? If the admin wants to see the teacher’s data, how does the app know which id to pull from?

Basically, do we need to make it so edits from either side (coach v. admin) update that information for both?  Or should it be limited so that a coach in a user account can make changes to teachers, and that updates in the admin portal.  But that means the admin portal is basically just used to summon up results and track who is working where and with whom, but not to add, transfer, or edit.
